extensions/ql-vscode/src/remote-queries/view/SortRepoFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { FilterIcon } from '@primer/octicons-react';
+import { ActionList, ActionMenu } from '@primer/react';
+
+export type Sort = 'name' | 'stars' | 'results';
+type SortBy = { name: string, sort: Sort }[];
```

Comment:
Shall we drop the array part here? I think it hurts readability.

```suggestion
type SortBy = { name: string, sort: Sort };
```

Reply:
I agree, that's confusing. I'll change that.

Reply:
This type is actually not needed. It's gone.